src/dotnet/commands/dotnet-tool/install/LocalizableStrings.resx

```diff
@@ -161,12 +161,19 @@ Tool '{1}' (version '{2}') was successfully installed.</value>
     <value>The settings file in the tool's NuGet package is invalid: {0}</value>
   </data>
   <data name="ToolInstallationFailed" xml:space="preserve">
-    <value>Tool '{0}' failed to install.</value>
+    <value>Tool '{0}' failed to install. This failure may have been caused by:
```

Comment:
@KathleenDollard @peterhuene this is the generic message no extra change other than the first sentence

Reply:
Isn't this only used for `ShellShimException` handling?  Seems like none of these explanations can occur for that exception.

Reply:
ShellShimException happens to be the only case. @peterhuene do you think we should revert to original? I am more worry about future usage.

Reply:
@peterhuene or we should just keep them the same?

Reply:
I think `ToolInstallationFailed` should stay the same and `ToolInstallationFailedWithExplicitVersionGuide` should use the text currently used here for `ToolInstallationFailed`.

Reply:
Fixed, and renamed as suggested